packages/@icon-magic/distribute/src/distribute-svg.ts

```diff
@@ -29,7 +29,7 @@ export async function distributeSvg(
   // Keep track of the sprites that have been created so we know when to create
   // a new one and when to append to an existing document
   const spriteNames: SpriteConfig = {};
-
+  const promises: any[] = [];
```

Comment:
Nitpick... more specific? `Promise<void>[]`? (Unless the promise resolves to something.)